Makefile

```diff
@@ -0,0 +1,20 @@
+
+
+monitor.zip: monitor
+	zip $@ $<
+
+monitor: lambda.go
+#	go get .
+	GOOS=linux GOARCH=amd64 go build -o $@ $<
+
+clean:
+	terraform destroy
+	rm -f init.done deploy.done monitor.zip monitor
+
+init.done:
+	terraform init
+	touch $@
+
+deploy.done: init.done agent-metrics.tf monitor.zip
+	terraform apply
+	touch $@
```

Comment:
Unrelated?

Reply:
The idea was to add a make file and the terraform file to make it easy to build and deploy the lambda, it may not match up with the rest of the project though.